backend/src/seed/factories/index.js

```diff
@@ -73,6 +73,7 @@ export default function Factory(options = {}) {
       const { factory, mutation, variables } = this.factories[node](args)
       if (factory) {
         this.lastResponse = await factory({ args, neodeInstance })
+        return this.lastResponse
```

Comment:
Ah, this is the messy part of the refactoring. The factories are currently in a dangerous in-between state. Some of them return a `graphql` mutation while the recently updated factories return a function.

I'm returning `this.lastResponse` here, because it's a neode type and we can call `relateTo` etc. on it. We cannot call this method on a plain JS object.